Plugins/Mute/MuteManager.cs

```diff
@@ -1,21 +1,67 @@
 ﻿using SharedLibraryCore;
+using SharedLibraryCore.Database.Models;
 
 namespace Mute;
 
 public class MuteManager
 {
-    public async Task<bool> Mute(GameEvent gameEvent)
+    private List<MuteStateMeta> _muteList = null!;
+    public bool HasMutes => _muteList.Any(x => x.Expiration > DateTime.UtcNow || x.Expiration == null);
+    public IEnumerable<MuteStateMeta> Mutes => _muteList.Where(x => x.Expiration > DateTime.UtcNow || x.Expiration == null);
+    public async void InitialiseMuteList() => _muteList = await Plugin.DataManager.ReadMuteList();
+
+    public async Task<bool> Mute(Server server, EFClient origin, EFClient target, DateTime? dateTime, string reason)
+    {
+        // Check if client has meta, if null, mute
+        var clientMetaState = await Plugin.DataManager.ReadPersistentData(target);
+        if (clientMetaState?.MuteState == MuteState.Muted) return false;
+
+        // Mute client
+        var client = server.GetClientsAsList().FirstOrDefault(x => x.NetworkId == target.NetworkId);
+        if (client is not null) await server.ExecuteCommandAsync($"muteClient {client.ClientNumber}");
```

Comment:
This could throw an exception if the rcon connection fails. 
In that case the state would never be written.
Maybe that's ok, but consider the expected behavior. Maybe we want to store the fact they were muted even if the rcon connection fails, so the next time they join it tries to mute them again.